lib/generators/geoblacklight/templates/settings.yml

```diff
@@ -160,6 +160,7 @@ OPACITY_CONTROL: &opacity_control
 
 LEAFLET:
   MAP:
+  ICON_DEFAULT_IMAGEPATH: ".."
```

Comment:
question: Is this a parameter that we need to make configurable to users? I guess I'm wondering what would make cause a implementation to change this value.

Reply:
Yeah, I was definitely under the influence of some stackoverflow comments where people were adopting different SVG iconography libraries. You might swap in a public dir path if you had a font awesome pro license and wanted their "duotone" look instead of Leaflet's default.

But, if you're willing to go that far, you're likely going to override GBL's viewer code... Maybe this is just unecessary? I'll think about it for a bit.

Reply:
If we had a use case / user request then I think we should consider it. I'm concerned that it's extra complexity in an already complex settings file.

Reply:
Sure. I've rolled that change back. We still need to set the path ourselves, otherwise Leaflet+Rails asset_url defaults the icon path to '//assets' and icons do not render.